dom0/sd-dom0-files.sls

```diff
@@ -104,6 +104,27 @@ dom0-install-securedrop-workstation-dom0-config:
       - file: dom0-workstation-rpm-repo
 {% endif %}
 
+dom0-environment-directory:
+  file.directory:
+    - name: /var/lib/securedrop-workstation/
+    - user: {{ gui_user }}
```

Comment:
Does it need to be owned by the gui_user? I think root would be better unless there's a specific reason